### ISSUE TITLE
rpcclient not a thing on OS X- SMB Spraying will not work

### ISSUE THREAD

**Issue by b4cktr4ck2** [Greenwolf/Spray#10]
There's, unfortunately, no brew cask (official or unofficial) for rpcclient, nor is there an official package that can be installed. As such, the SMB portion of the script sadly will not work on a Mac.

If someone finds a version of rpcclient for Darwin or a workaround I'll gladly close this- I spent a few hours trying to find a copy that would work.

**Comment by Greenwolf** [comment]
Thank you @b4cktr4ck2 for bringing this to my attention. I've updated the readme with additional info saying it is not possible to run smb mode on macOS. I've added this as a feature request, and will leave it open. 

If anyone finds a way to install rpcclient on OSX, please let me know!

**Comment by Esonhugh** [comment]
seems rpcclient is in the package called samba. 
installing by command `brew install samba`? 
and you will got smbclient/rpcclient binary.

**Comment by MelForze** [comment]
Yes it's true, you can grab the binary from the path (I had one) `/opt/homebrew/Cellar/samba/4.20.1/bin/rpcclient` and everything works fine. Add it to path and there is no problem.

**Comment by adibna** [comment]
for me, the /opt/homebrew/Cellar/samba/4.20.1/bin don`t have the "rcclient"

**Comment by MelForze** [comment]
> for me, the /opt/homebrew/Cellar/samba/4.20.1/bin don`t have the "rcclient"

Maybe try to reinstall and check again ?

**Comment by Esonhugh** [comment]
> > for me, the /opt/homebrew/Cellar/samba/4.20.1/bin don`t have the "rcclient"
> 
> Maybe try to reinstall and check again ?

It really no rcclient :) but rpcclient
